cypress/e2e/spec.cy.js

```diff
@@ -92,4 +93,133 @@ describe("Readim 첫 진입시 초기화면", () => {
         expect(wpm).to.equal("202");
       });
   });
+
+  it("아티클 카드가 없는 경우 컨셉 문구가 순환하고 있어야 한다.", () => {
+    cy.visit("/");
+    cy.clock();
+
+    const conceptTexts = ["절약할", "파악할", "활용할", "계획할"];
+
+    conceptTexts.forEach((text, index) => {
+      if (index > 0) {
+        cy.tick(1900);
+      }
+      cy.getBySel("test-conceptText").should("have.text", text);
+    });
+  });
+
+  it("URL입력 전에는 articleCard가 하나도 없어야 한다.", () => {
+    cy.visit("/");
+    cy.getBySel("test-cardContainer").children().should("not.exist");
+  });
+
+  describe("정확한 URL을 입력하는 경우", () => {
+    const correctUrl = "https://developer.mozilla.org/ko/docs/Web/JavaScript";
+
+    it("입력창에 URL을 입력한 후 Enter를 누르면 API를 요청해야한다.", () => {
+      requestURL(correctUrl);
+    });
+
+    it("API요청 응답이 온 경우, 입력창이 초기화 되고 아티클 카드가 추가돼야 한다.", () => {
```

Comment:
```suggestion
    it("API요청 응답이 온 경우, 입력창이 초기화 되고 아티클 카드가 추가돼야 합니다.", () => {
```
테스트 케이스 제목의 어미를 `~합니다` 또는 `~한다` 로 통일하는 것은 어떨까요?